flowdb/bin/build/0030_schema_interactions.sql

```diff
@@ -0,0 +1,158 @@
+/*
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+/*
+Interaction -------------------------------------------------------
+
+This schema contains tables related to interactions between 
+subscribers.
+
+  - subscriber:                 subscriber encountered in the event 
+                                data.
+  - locations:                  contains a new row for each time a cell
+                                moves
+  - receiving_parties:
+  - event_supertable:           contains a row per call/mds/sms event. The
+                                event_type field may be any of 1. calls, 2. sms, 3. mds and
+                                fianlly 4. topup
+  - subscriber_sightings:       contains a row per subscriber sighting event
+  - calls:                      stores the additional call type data
+  - sms:                        stores the additional sms type data
+  - mds:                        stores the additional mobile data type data
+  - topup:                      stores the additional topup type data
+-----------------------------------------------------------
+*/
+CREATE SCHEMA IF NOT EXISTS interactions;
+
+    CREATE TABLE IF NOT EXISTS interactions.subscriber(
+
+        id                      BIGSERIAL PRIMARY KEY,
+        msisdn                  TEXT,
+        imei                    TEXT,
+        imsi                    TEXT,
+        tac                     BIGINT REFERENCES infrastructure.tacs(id)
+        );
+
+    CREATE INDEX ON interactions.subscriber (id);
+    CREATE INDEX ON interactions.subscriber (id, msisdn);
+
+
+
+    CREATE TABLE IF NOT EXISTS interactions.locations(
+
+        cell_id                 BIGSERIAL PRIMARY KEY,
+        site_id                 TEXT,
+        mno_cell_code           TEXT
```

Comment:
I think this should really be a FK referencing the cells table. That's a little awkward, because the cells table is PKd on the mno id and version, so we might want to adjust that to be keyed on an id we generate and uniqued on those fields?